server.js

```diff
@@ -0,0 +1,27 @@
+var express = require('express'),
```

Comment:
`"use strict";` is a nice thing to include at the top of every file, as it forces V8 into strict mode (turns some runtime warnings into parse-time errors).

Consistency in quoting is good--I'm partial to double quotes for no particularly good reason (single quotes are characters!).